Dockerfile

```diff
@@ -37,9 +53,8 @@ COPY --from=operator-build /usr/lib/libgpgme.so.* /usr/lib/
 COPY --from=k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.1 /csi-node-driver-registrar /usr/local/bin
 COPY --from=k8s.gcr.io/sig-storage/livenessprobe:v2.7.0 /livenessprobe /usr/local/bin
 
-# csi depdenencies
-COPY --from=dependency-src /bin/mount /bin/umount /bin/
-COPY --from=dependency-src /lib64/libmount.so.1 /lib64/libblkid.so.1 /lib64/libuuid.so.1 /lib64/
+# depdenencies
+COPY --from=dependency-src /mnt/installfs/ /
```

Comment:
can we try to copy files explicitly same as before instead of just copying the whole directory?

Reply:
see above